practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -67,10 +88,22 @@ const FormValidation = (props) => {
       setErrors(newErrors);
 
       if (Object.keys(newErrors).length === 0) {
-        onShowForm();
+        const response = await createCustomerService(formData);
```

Comment:
how about update customer?

Reply:
When I open a form to edit customer, I will check if having ID, it will make a update feature